README.md

```diff
@@ -1,76 +1,80 @@
+  - Set breakpoints, inspect variables, and perform other debugging tasks with integrated debugging support and without having to manually set up configuration. [Read the docs](https://cloud.google.com/code/docs/vscode/debug)
+  - Make easy work of setting up and customizing Kubernetes configuration files with Cloud Code’s YAML authoring assistance with out-of-the-box solutions for common schema, support for Custom Resources (CRDs) like Istio and Knative, smart completions, syntax coloring, documentation on hover, and linting support. [Read the docs](https://cloud.google.com/code/docs/vscode/yaml-editing)
+</details>
+
+## Cloud Run development
+
+Create and deploy a new service in minutes with Cloud Code’s equally robust Cloud Run support, monitoring your service’s progress with the Cloud Run Explorer and service logs accessible in the Log Viewer. If you’d prefer a local development workflow, you can also develop and debug a service locally with the built-in Cloud Run emulator .
+
+<details>
+  <summary>Read more</summary>
+
+  ### Highlights:
+  - Deploy a service to Cloud Run, customizing your deployment platform and build settings along the way, from within your IDE. [Read the docs](https://cloud.google.com/code/docs/vscode/deploying-a-cloud-run-app)
+  - Locally debug your service via the Cloud Run emulator and perform tasks you normally do when debugging local code as you develop your app. With Cloud Code’s fast iterative development, you can automatically redeploy changes to the emulator as you make them. [Read the docs](https://cloud.google.com/code/docs/vscode/debugging-a-cloud-run-app)
+  - Monitor the status of your Cloud Run services as well as their revisions and essential properties with the Cloud Run Explorer. [Read the docs](https://cloud.google.com/code/docs/vscode/cloud-run-explorer)
+</details>
+
+## Containerization made easy
+Create secure, production-ready container images from source code without having to worry about a Dockerfile with Cloud Code’s built-in support for Google Cloud Buildpacks. You get to focus on building your application, not containerizing it. 
+
+## Built-in Secret Manager support
+Protect sensitive information and keep your app secure with Cloud Code’s integrated Secret Manager support. You can create, view, update, and use secrets in the Secret Manager view without having them in your codebase. [Read the docs](https://cloud.google.com/code/docs/vscode/secret-manager)
+
+## Google Cloud APIs at your fingertip
+Browse available Cloud APIs, enable services, and install and learn how to integrate client libraries in your app without leaving your IDE and breaking your development flow, all with the API library browser. [Read the docs](https://cloud.google.com/code/docs/vscode/client-libraries)
+
+- Get straight to developing with Cloud Code’s simplified authentication workflow that uses your Google Cloud credentials.
```

Comment:
You can drop lines 68-73, since it's been added to the first list up top.

Reply:
Done. Thanks.